csrc/runtime/fusion_kernel_runtime.cpp

```diff
@@ -461,6 +461,8 @@ void FusionKernelRuntime::compileFusionParallel(KernelArgumentHolder args) {
               val->isA<TensorView>(),
               "Output must be a TensorView but got ",
               val);
+          struct AliasInfo alias_info = segmented_fusion_->completeFusion()->getOutputAlias(val);
+          NVF_ERROR(alias_info.type == AllocationType::New, "Output ", val->toString(), " must not be an alias, got ", alias_info.toString());
```

Comment:
```suggestion
          NVF_ERROR_EQ(alias_info.type, AllocationType::New, "Output ", val->toString(), " must not be an alias, got ", alias_info.toString());
```
https://github.com/NVIDIA/Fuser/blob/a8ad0bc05451ab1dca0892558e5dca73fb89be6a/csrc/exceptions.h#L275 is the benefit

Reply:
Got it. 

> AllocationType is just an enum so it doesn't have << implemented.

It should and I think it's as simple as making https://github.com/NVIDIA/Fuser/blob/a8ad0bc05451ab1dca0892558e5dca73fb89be6a/csrc/fusion.h#L108-L117 an operator<<. However, I'm happy to take it as a separate PR